--- origsrc/xc/lib/Xaw3d/Imakefile	2003-03-08 08:55:18.000000000 -0600
+++ src/xc/lib/Xaw3d/Imakefile	2007-01-10 14:09:16.734375000 -0600
@@ -5,11 +5,11 @@
 XCOMM Build options. Have you read the README.XAW3D file yet?
 XCOMM
 XCOMM For color pixmaps, define MULTIPLANE_PIXMAPS:
-#undef MULTIPLANE_PIXMAPS
+#define MULTIPLANE_PIXMAPS
 XCOMM For grayed stipple shadows, define GRAY_BLKWHT_STIPPLES:
 #define GRAY_BLKWHT_STIPPLES
 XCOMM For scrollbars with arrows, define ARROW_SCROLLBARS:
-#undef ARROW_SCROLLBARS
+#define ARROW_SCROLLBARS
 
 #define DoNormalLib NormalLibXaw
 #define DoSharedLib SharedLibXaw
@@ -22,7 +22,7 @@
 #define IncSubSubdir Xaw3d
 
 XCOMM When building outside an X11 source tree:
-XCOMM EXTRA_INCLUDES = -I.
+EXTRA_INCLUDES = -I../..
 
 #ifdef SharedXawReqs
 REQUIREDLIBS = SharedXawReqs
--- origsrc/xc/lib/Xaw3d/app-defaults/Fig	2003-03-09 12:22:02.000000000 -0600
+++ src/xc/lib/Xaw3d/app-defaults/Fig	2007-01-10 13:44:56.953125000 -0600
@@ -1,5 +1,5 @@
 
-#include "/usr/X11R6/lib/X11/app-defaults/Fig-color"
+#include "/etc/X11/app-defaults/Fig-color"
 
 ! This bunch is for Xaw3d...
 #include "xawthreedwidget"
